packages/did/src/Did.chain.ts

```diff
@@ -249,6 +251,13 @@ export async function getDeleteDidExtrinsic(): Promise<Extrinsic> {
   return api.tx.did.delete()
 }
 
+export async function generateReclaimDepositExtrinsic(
```

Comment:
The other functions returning `Extrinsic` instead of `SubmittableExtrinsic` is just a choice from the developer who did that code. `SubmittableExtrinsic` extends `Extrinsic`, that's why it works.
I would prefer renaming and changing the return type to `Promise<Extrinsic>`

In general regarding `get` vs `generate` in this file, I interpret it, that the get functions basically just map to the correct api call, whereas generate does more stuff needed to generate a specific api call.